### ISSUE TITLE
Allow shipping a default configuration under Linux

### ISSUE THREAD

**Issue by stalebrim** [nuttyartist/notes#435]
Hello,
Could it be possible to add a /usr configuration file that the application would source its default configuration from on Linux systems? This would allow package maintainers to e.g. use native header by default.

**Comment by nuttyartist** [comment]
What do you mean by native header? Native window frame? What other examples can our app get from the /usr configuration file?

**Comment by bjorn** [comment]
I'm not sure why the choice of default native frames should be made by package maintainers. Why should Notes use native frames by default on some distros but not on others? Maybe we should instead re-evaluate whether it is a good idea to not use native headers by default on Linux in general, or fix related issues, if those are the reason to use native frames.

**Comment by stalebrim** [comment]
> Maybe we should instead re-evaluate whether it is a good idea to not use native headers by default on Linux in general, or fix related issues, if those are the reason to use native frames.

I think it's a matter of having the app integrate well to the desktop. The native frame is in line with other window decorations and their operation.

**Comment by nuttyartist** [comment]
@bjorn I guess the point is that different distros would be able to customize, theme, and set certain defaults that go well with their own style. I think it's a welcome suggestion. I also think there should be some form of `reset to default manufacture settings` that will allow users to still revert to the original experience.

**Comment by stalebrim** [comment]
> @bjorn I guess the point is that different distros would be able to customize, theme, and set certain defaults that go well with their own style.

Yes. That's what I was after. Apologies for not putting it into words in a clear way. My impression is that it usually works like that there's a distro-provided configuration file in /usr, which then can be overridden by one in /etc or user's own configuration in their ~/.config.
